cmd/dcrtime/dcrtime.go

```diff
@@ -33,18 +33,20 @@ const (
 var (
 	testnet   = flag.Bool("testnet", false, "Use testnet port")
 	debug     = flag.Bool("debug", false, "Print JSON that is sent to server")
-	printJson = flag.Bool("json", false, "Print JSON response from server")
+	printJSON = flag.Bool("json", false, "Print JSON response from server")
```

Comment:
This should be a separate PR but I'll let it slide for now.

Reply:
🙈